### DISCUSSION TITLE
treetable的子节点使用exportTemplet函数导出异常

### DISCUSSION THREAD

**Discussion by hjingsuper** [layui/layui#2030] [讨论]
treetable的子节点使用exportTemplet函数导出无数据。treetable没有子节点或table 则没有此问题，可以顺利使用exportTemplet函数
[示例](https://codepen.io/hjingsuper/pen/yLWJPoW)
![image](https://github.com/layui/layui/assets/36070786/bcb3ef3c-1238-4c61-8e4b-2d0d727344ce)

操作步骤：
1. treetable初始化
2. 双击行，选择【性别】
3. 导出

**Comment by sentsim** [comment]
关联：#2034
